docs/PUBLISHING.md

```diff
@@ -92,14 +92,14 @@ access resin.io's production downloads S3 bucket.
 Run the following command to publish a specific file:
 
 ```sh
-./scripts/publish/aws-s3.sh <file>
+./scripts/publish/aws-s3.sh -f <file> -b <bucket> -v <version> -t <production|development>
```

Comment:
Given that `<bucket>` never changes, should this be documented as `... -b resin-production-downloads ...` ?

Or maybe it should just be hardcoded back inside `aws-s3.sh` rather than being a command-line option? (much like the way that `bintray-debian.sh` hardcodes `PACKAGE_COMPONENT=etcher`)